ERROR in ./~/react-big-calendar/lib/TimeColumn.js

Hi, since the 0.10 update, I got this error.

```
ERROR in ./~/react-big-calendar/lib/TimeColumn.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./TimeSlotGroup.jsx in .../my-project/node_modules/react-big-calendar/lib
 @ ./~/react-big-calendar/lib/TimeColumn.js 17:21-51
```

I did not try (for now) to implement custom views, I just made the update.

It may come from #107

--- a/src/TimeColumn.jsx
+++ b/src/TimeColumn.jsx
@@ -3,7 +3,7 @@ import cn from 'classnames';
 
 import dates from './utils/dates';
 
-import TimeSlotGroup from './TimeSlotGroup.jsx'
+import TimeSlotGroup from './TimeSlotGroup'
 
 export default class TimeColumn extends Component {
   static propTypes = {

